prime-router/src/main/kotlin/azure/HttpUtilities.kt

```diff
@@ -17,7 +17,7 @@ import java.time.ZoneOffset
 import java.time.format.DateTimeFormatter
 
 enum class ReportStreamEnv(val urlPrefix: String) {
-    TEST("https://pdhtest-functionapp.azurewebsites.net"),
+    TEST("https://test.prime.cdc.gov"),
```

Comment:
Update the Environments.kt file as well or use the Environments.kt info.  This enum will be replaced/removed soon with Environment.kt